src/main/java/com/iridium/iridiumskyblock/commands/DepositCommand.java

```diff
@@ -80,9 +82,16 @@ public boolean execute(CommandSender sender, String[] args) {
      */
     @Override
     public List<String> onTabComplete(CommandSender commandSender, org.bukkit.command.Command command, String label, String[] args) {
-        return IridiumSkyblock.getInstance().getBankItemList().stream()
-            .map(BankItem::getName)
-            .collect(Collectors.toList());
+        if (args.length == 2) {
+            List<String> possible = IridiumSkyblock.getInstance().getBankItemList().stream()
+                    .map(BankItem::getName)
+                    .collect(Collectors.toList());
+
+            List<String> actualTab = possible.stream().filter(s -> s.toLowerCase(Locale.ROOT).startsWith(args[1].toLowerCase(Locale.ROOT))).collect(Collectors.toList());
```

Comment:
it could be String#contains instead startsWith

Reply:
I'm pretty sure that we use `startsWith()` everywhere else too

Reply:
I'm pretty sure that we use `startsWith()` everywhere else too